package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "description": "Project Platform Health",
   "scripts": {
-    "build": "neutrino build",
-    "start": "neutrino start",
-    "start:local": "BACKEND=http://localhost:3000 neutrino start",
-    "lint": "neutrino lint",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open",
+    "start:local": "BACKEND=http://localhost:3000 webpack-dev-server --mode development --open",
+    "test": "jest",
+    "lint": "eslint --cache --ext mjs,jsx,js --format codeframe src",
     "precommit": "lint-staged",
-    "prepush": "neutrino test",
-    "coverage": "rm -rf coverage && nyc --all neutrino test && nyc report -r lcov",
+    "prepush": "npm run test",
```

Comment:
This and the others should probably be `yarn run` instead of npm? (yarn also supports the abbreviated form that omits `run`, eg `yarn test`)